addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,52 +1,13 @@
-.o_dialog {
-  > .modal {
-    display: block;
-  }
-}
-
-.modal.o_technical_modal {
-  .modal-content {
-    border-radius: 0;
-
-    .modal-header .modal-title {
-      word-break: break-word;
-    }
-
-    .modal-footer {
-      flex-wrap: wrap;
-      text-align: left;
-      justify-content: flex-start;
+.modal.o_technical_modal .modal-footer {
+    button {
+        margin: 0 map-get($spacers, 1) 0;
```

Comment:
The previous implementation used `margin-bottom: 0.5rem;` (https://github.com/odoo/odoo/pull/83128/files#diff-a05dbc33bacc0af5e27886ecba2e1cbc70cb2b67290d29a1ac445df66f5b5a1bL28).
Is this no longer necessary?

Reply:
It is actually. 🙏 
Top and bottom is now getting a `map-get($spacer, 1)`  so we'll have an equal spacing inside the footer of the modal (rather than having a bigger spacing only at the bottom), and of course we'll have the require spacing between buttons if it goes into two lines.